extensions/wikia/Wall/Wall.class.php

```diff
@@ -169,7 +169,14 @@ public function getUrl() {
 		wfProfileIn(__METHOD__);
 		$title = Title::newFromText( $this->getUser()->getName(), NS_USER_WALL );
 		wfProfileOut(__METHOD__);
-		return $title->getFullUrl();
+		if ( $title instanceof Title ) {
+			return $title->getFullUrl();
+		} else {
```

Comment:
`else` is not necessary since returning on previous line and following code wont need to be indented.